esp-wifi/src/timer/xtensa.rs

```diff
@@ -47,9 +54,19 @@ pub fn setup_multitasking() {
     }
 }
 
+pub fn disable_multitasking() {
+    let config = unsafe { xtensa_lx::interrupt::enable() };
+    xtensa_lx::interrupt::disable_mask(!config);
+    xtensa_lx::interrupt::disable_mask(
+        1 << 29 // Disable Software1
```

Comment:
I think I misread this in my first review (sorry!) - I think this should just be `disable_mask(1 << 29 // Disable Software1)`